### ISSUE TITLE
Suggest features components selection from PCA method in multi-scale fusion.

### ISSUE THREAD

**Issue by linjing-lab** [MolecularMaterials/AL-HTE-Electrolyte#1]
In multi-scale modeling, LUMO_sol and HOMO_sol terms are electronic microscale features calculated by DFT, and using only normalization and PCA of In [18] of BTZ-binarySolvents-productionRun.ipynb for automatic feature dimensionality reduction will ignore the importance of prior knowledge in feature representation.
```python
n_PC = 5  # Set number of principle components
st = StandardScaler()
Xdata = st.fit_transform(X)  # Normalize feature vectors

pca = PCA(n_components=n_PC)
Xdata = pca.fit_transform(Xdata)  # Transform feature vectors to PCs
```
This method leverages gpnetwork training in robotic platform, but ignore each feature contribution, for example, HeavyAtomCount_solv consists of microscale feature in molecular conformational properties. A simple MLP(11, 5, 1) of 5 hidden size network will represents features into latent feature extraction and predict regression task in this binarySolvents production. Nerual network line with extract important features at In [6] of [Molecular prediction with SELFIES representation](https://github.com/linjing-lab/easy-pytorch/blob/main/released_box/TP53_selfies.ipynb) presents high precision with hidden size less than total one_hot sequence representation.
